mne/viz/evoked.py

```diff
@@ -1191,3 +1191,342 @@ def plot_evoked_joint(evoked, times="peaks", title='', picks=None,
+    """
+    from ..evoked import Evoked
+    import matplotlib.pyplot as plt
+
+    # set up labels and instances
+    if isinstance(evokeds, Evoked):
+        evokeds = dict(Condition=evokeds)
+
+    if isinstance(evokeds, dict):
+        conditions = sorted(list(evokeds.keys()))
+    else:
+        evokeds = dict((str(ii + 1), evoked)
+                       for ii, evoked in enumerate(evokeds))
+        conditions = sorted(list(evokeds.keys()))
+
+    if conditions is None:
+        conditions = sorted([str(ii) for ii in evokeds])
+    if isinstance(conditions, string_types):
+        conditions = [conditions]
+
+    # get and set a few limits and variables (times, channels, units)
+    example = (evokeds[conditions[0]]
+               if isinstance(evokeds[conditions[0]], Evoked)
+               else evokeds[conditions[0]][0])
+    times = example.times
+    tmin, tmax = times[0], times[-1]
+
+    # deal with picks: infer indices and names
+    if picks is None:
+        if ymin is None:
+            ymin = 0
+        ch_type = channel_type(example.info, 0)
+        ch_names = ['GFP']
+    else:
+        if isinstance(picks, int):
+            ch_names = [example.ch_names[picks]]
+            picks = [picks]
+        elif isinstance(picks, string_types):
+            ch_names = [picks]
+            picks = [example.ch_names.index(picks)]
+        elif isinstance(picks[0], int):
+            ch_names = [example.ch_names[pick] for pick in picks]
+        elif isinstance(picks[0], string_types):
+            ch_names = picks[:]
+            picks = [example.ch_names.index(pick) for pick in picks]
+        else:
+            raise ValueError("`picks` must be int, a list of int, "
+                             "str, a list of str, or None, not " +
+                             str(type(picks)))
+        ch_type = channel_type(example.info, picks[0])
+    scaling = _handle_default("scalings")[ch_type]
+
+    if ch_type == 'grad' and picks is not None:
+        from ..channels.layout import _merge_grad_data, _pair_grad_sensors
+        picked_chans = []
+        pairpicks = _pair_grad_sensors(example.info, topomap_coords=False)
+        for pick in picks:
+            picked_chans.append(pairpicks.index(pick))
+            picked_chans.append(pairpicks.index(pick - 1)
+                                if pick % 2 else pairpicks.index(pick + 1))
+        picks = list(sorted(set(picked_chans)))
+        ch_names = [example.ch_names[pick] for pick in
+                    [pairpicks[p_] for p_ in picks]]
+        if ymin is None:
+            ymin = 0
+
+    # deal with dict/list of lists, and calculate the CI
+    if not isinstance(evokeds[conditions[0]], Evoked):
+        from ..evoked import combine_evoked
+        if not isinstance(evokeds[conditions[0]][0], Evoked):
+            raise ValueError("evokeds must be an `mne.Evoked` "
+                             "or of a collection of `mne.Evoked`s")
+
+        if ci and picks is not None:
```

Comment:
Hm ... So either True should default to .95, or it should be enforced float only?